ccloud/docs/beginner-cloud.rst

```diff
@@ -472,7 +466,7 @@ Run a Java producer to show a prefixed ACL
 
       ccloud kafka acl list --service-account 104349
 
-   Your output should resemble:
+   Your output should resemble below. Note the ACL Type is ``PREFIXED``.
```

Comment:
```suggestion
   Your output should resemble:
```

Reply:
It's helpful to the user to call out `PREFIXED`